content-src/asrouter/rich-text-strings.js

```diff
@@ -7,6 +7,7 @@ import {MessageContext} from "fluent";
  */
 const RICH_TEXT_CONFIG = {
   "text": ["text", "scene1_text"],
+  "scene2_text": "scene2_text",
```

Comment:
So this was a weird one. This way of expressing multiple possibilities with `Array` doesn't work if 2 or more options will be present. If you have both `scene1` and `scene2` when it looks for `scene1` text, because in the code we do `.pop()` it will pick from the end `scene2` and offer that translation option. Essentially showing `scene2` text twice.

Reply:
O_o